quantum/nodes/grovers/locales/en-US/grovers.html

```diff
@@ -0,0 +1,55 @@
+<script type="text/html" data-help-name="grovers">
+  <p>
+    This node applies Grover's algorithm to find specified bit string value in the search space
```

Comment:
@ttoktassynov I would appreciate if you could take care of the node's documentation since I am not familiar with it.

Reply:
I have added documentation